From 8f8271e1085f6f8e306c689d6a56b0bb8d076444 Mon Sep 17 00:00:00 2001
From: Lars Kiesow <lkiesow@uos.de>
Date: Thu, 20 May 2021 12:07:46 +0200
Subject: [PATCH] Update CXF

This updates CXF to version 3.4.3, fixing among other things
[CVE-2019-12419], [CVE-2019-12423], [CVE-2021-22696], [CVE-2019-12406],
[CVE-2020-13954], [CVE-2019-17573], [CVE-2020-1954].

The security issues should not be problematic for Opencast and we can
treat this as a regular update.
---
 modules/security-aai/pom.xml | 2 +-
 pom.xml                      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/security-aai/pom.xml b/modules/security-aai/pom.xml
index 0aeb280ccc8..4a89786cfc6 100644
--- a/modules/security-aai/pom.xml
+++ b/modules/security-aai/pom.xml
@@ -106,7 +106,7 @@
               <failOnWarning>true</failOnWarning>
               <ignoredUsedUndeclaredDependencies>
                 <ignoredUsedUndeclaredDependency>junit:junit</ignoredUsedUndeclaredDependency>
-                <ignoredUsedUndeclaredDependency>javax.annotation:javax.annotation-api</ignoredUsedUndeclaredDependency>
+                <ignoredUsedUndeclaredDependency>jakarta.annotation:jakarta.annotation-api</ignoredUsedUndeclaredDependency>
                 <ignoredUsedUndeclaredDependency>org.springframework:spring-expression</ignoredUsedUndeclaredDependency>
                 <ignoredUsedUndeclaredDependency>org.springframework:spring-core</ignoredUsedUndeclaredDependency>
                 <ignoredUsedUndeclaredDependency>org.springframework.security:spring-security-core</ignoredUsedUndeclaredDependency>
diff --git a/pom.xml b/pom.xml
index 7606df7822d..5287e807693 100644
--- a/pom.xml
+++ b/pom.xml
@@ -25,7 +25,7 @@
     <commons-io.version>2.8.0</commons-io.version>
     <commons-lang3.version>3.11</commons-lang3.version>
     <commons-text.version>1.9</commons-text.version>
-    <cxf.version>3.3.3</cxf.version>
+    <cxf.version>3.4.3</cxf.version>
     <eclipselink.version>2.7.5</eclipselink.version>
     <failureaccess.version>1.0.1</failureaccess.version>
     <freemarker.version>2.3.30</freemarker.version>
